### PR TITLE
Run tests against rhche. Start refactoring of fucntional-tests.

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: Radim Hopp <rhopp@redhat.com>

### CODE REVIEW

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
@ScrewTSW pls review

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
Tests on prod are passing.

File: tests/src/test/java/redhat/che/functional/tests/fragments/TabsPanel.java [hunk 1]
```diff
@@ -4,41 +4,54 @@
 import org.jboss.arquillian.graphene.findby.FindByJQuery;
```
Comment (ScrewTSW) [inline tests/src/test/java/redhat/che/functional/tests/fragments/TabsPanel.java]: Chyba licencia :D

File: tests/src/test/java/redhat/che/functional/tests/suites/StableTestSuite.java [hunk 2]
```diff
@@ -28,11 +28,11 @@
...
-    PomTestCase.class,
-    AnalyticsErrorMarkersTestCase.class,
-    MavenTestCase.class,
-    WorkspacesTestCase.class,
-    PackageJsonTestCase.class
+//    PomTestCase.class,
+//    AnalyticsErrorMarkersTestCase.class,
+//    MavenTestCase.class,
+//    WorkspacesTestCase.class,
+//    PackageJsonTestCase.class
```
Comment (ScrewTSW) [inline tests/src/test/java/redhat/che/functional/tests/suites/StableTestSuite.java]: Ostatne este nefunguju?

File: tests/src/test/java/redhat/che/functional/tests/fragments/TabsPanel.java (hunk 1 above)
Comment (rhopp) [reply tests/src/test/java/redhat/che/functional/tests/fragments/TabsPanel.java]: Resolved.

**Pull request merged**